plugin/evm/validators_cache_test.go

```diff
@@ -0,0 +1,87 @@
+package evm
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type testData map[uint64]map[ids.ShortID]uint64
+
+func (testData) ByEpoch(uint64) (map[ids.ShortID]uint64, error) {
+	return map[ids.ShortID]uint64{}, nil
+}
```

Comment:
Because I only need to satisfy the interface.
And the data type I need to create the test is not the returned ```map[ids.ShortId]uint64``` but ```map[uint64]map[ids.ShortId]uint64```